### ISSUE TITLE
i can't restore my iPhone 5s 

### ISSUE THREAD

**Issue by EdwardTCM** [LukeZGD/iOS-OTA-Downgrader#64]
everytime i try to downgrade and get to pwnDFU it fails.
******* iOS-OTA-Downgrader ******* 
   Downgrader script by LukeZGD    

[Log] Finding device in normal mode... 
[Log] Finding device in DFU/recovery mode... 
* Platform: linux  
* HardwareModel: n53ap 
* ProductType: iPhone6,2 
* ProductVersion: Unknown 
* UniqueChipID (ECID): 4080975764764 

[Log] Option: Downgrade 
[Log] Device in recovery mode detected. Get ready to enter DFU mode 
[Input] Select Y to continue, N to exit recovery (Y/n)  y
* Hold POWER and HOME button for 8 seconds. 
08 07 06 05 04 03 02 01 
* Release POWER and hold HOME button for 8 seconds. 
08 07 06 05 04 03 02 01 
[Log] Device in DFU mode detected. 
[Log] Entering pwnDFU mode with ipwndfu... 
*** checkm8 exploit by axi0mX ***
*** modified version by Linus Henze and synackuk ***
Found: CPID:8960 CPRV:11 CPFM:03 SCEP:01 BDID:02 ECID:000003B62D1C5D1C IBFL:1C SRTG:[iBoot-1704.10]
Traceback (most recent call last):
  File "ipwndfu", line 68, in <module>
    checkm8.exploit()
  File "/home/edward/iOS-OTA-Downgrader/resources/ipwndfu/checkm8.py", line 529, in exploit
    if 'PWND:[checkm8]' not in device.serial_number:
  File "/home/edward/iOS-OTA-Downgrader/resources/ipwndfu/usb/core.py", line 830, in serial_number
    self._serial_number = util.get_string(self, self.iSerialNumber)
  File "/home/edward/iOS-OTA-Downgrader/resources/ipwndfu/usb/util.py", line 314, in get_string
    raise ValueError("The device has no langid")
ValueError: The device has no langid

[Error] Failed to enter pwnDFU mode. Please run the script again 
* ./restore.sh Downgrade 

can someone help me here please

**Comment by LukeZGD** [comment]
There is not much that I can do for improved success rates with ipwndfu

See other notes in the readme

**Comment by r34per101** [comment]
I got it working on a 2009 MacBook rubbing high Sierra. Running the script straight from main folder would always fail. Can’t remember where I read it but I got it working by opening a new terminal window and cd in to the ipwndfu folder and running the ipwndfu command there. Took several attempts but it worked and then went back to the downgrade terminal window and ran the restore.SH again and it worked
